source/systems/pan/index.rst

```diff
@@ -16,12 +16,608 @@ Intel processors, from Sandy Bridge to Coffee Lake, each has
+.. code-block:: shell
+
+   #PBS                    #SBATCH (long)                  #SBATCH (short)
+   ----                    --------------                  ---------------
+   -N jobname              --job-name=jobname              -J jobname
+
+   -l size=1                 default
+   -l nodes=1:ppn=1        --ntasks=1                      -n 1
+   -l nodes=1:ppn=2        --ntasks=2                      -n 2
+   -l walltime=10:00:00    --time=10:00:00                 -t 10:00:00
+
+   -d dir                  --chdir=dir                     -D dir
+
+   -o dir                  --output=dir/%x.o%j             -o dir/%x.o%j
+   -o file                 --output=file                   -o file
+   -j oe                     default
+
+   -r y                      default
+   -r n                    --no-requeue
+
+   -q analysis             --partition=analysis            -p analysis
+   -q bigmem               --constraint=bigmem             -C bigmem
+   -q bigvftmp             --constraint=bigvftmp           -C bigvftmp
+
+   -v VAR=value            --export=VAR=value
+   -A gfdl_x               --account=gfdl_x                -A gfdl_x
+
+   -m abe                  --mail-type=all
+   -m a                    --mail-type=fail
+   -M my.name@noaa.gov     --mail-user=my.name@noaa.gov
+
+   -S /bin/csh               none
+
+Order of precedence is:
+
+lowest:  #SBATCH lines in jobscript
+middle:  SBATCH_* environment variables
+highest: sbatch command line options
+
+$TMPDIR can be used without change. It is created and removed at job start and
+end. On Slurm, TMPDIR is /vftmp/My.Name/job12345. If you use $TMP, change it to
+$TMPDIR.
+
+Gotchas
+-------
+
+#. If -o is a directory, the job submission succeeds, the job is scheduled, and
+immediately fails with reason NonZeroExitCode.
+
+3. If -D dir is not specified, the job's working directory is the submission
+directory. If this directory does not exist on the execution host, Slurm does
+"cd /tmp" and runs the job.
+
+#. Slurm redirects standard output and standard error to the logfile pointed to
+   in the header only after a line is finished executing. If you need a
+   heartbeat to monitor script progress, consider using another meachanism.
+
+#. If slurm scripts do not end with a POSIX-standard new line character, the
+   last line of the script will not execute. Please note that this is not an
+   issue if you are editing your script; this is a possible issue if you
+   auto-generate code  for batch submission outside of FRE. (`Reference
+   <https://thoughtbot.com/blog/no-newline-at-end-of-file>`_
+
+
+Analysis Software
+=================
+
+To access most analysis software, you must use the "module" command, described
+below. Only matlab, idl, and mathematica are accessible without loading a
+module.
+
+Most GFDL software on PP/AN (and workstations) is managed by
+Spack, which facilitates easier, automated, and more frequent software updates.
+(GFDL Spack-managed software environment wiki)
```

Comment:
"(GFDL Spack-managed software environment wiki)" --> is this supposed to be a link to something?